### ISSUE TITLE
发送消息报错退出

### ISSUE THREAD

**Issue by 763147503** [botorange/wechaty-puppet-padpro#64]
> 重要：请不要删除模板自行填写，所有不按照模板填写的issue，我们将不会处理。

## 0. 发 Issue 指南

1. 请运行下面的命令，看问题是否可以被解决：
```
rm -rf package-lock.json
rm -rf node_modules
npm install
```

2. 请在 [FAQ 清单](https://docs.chatie.io/v/zh/faq) 看是否已有解决办法。

3. 请先在issue 中搜关键信息，确认你要发的内容和之前的issue 不重复。

## 1. 提供你的包版本信息
- wechaty 版本：当前最新版本
- wechaty-puppet-padpro 版本："0.0.47"
- node 版本： v10.15.0
- 你使用的操作系统：centos7

## 2. Bug 描述
报错：
21:29:01 INFO WechatGateway sendShort() failed for error: sendShort receive unknown package: [126] 7e0ffffffff300000000b142080d000000004ad5090eJ�e0008aD��], retry the api.
21:29:01 INFO WechatGateway sendShort() failed for error: sendShort receive unknown package: [126] 7e0ffffffff300000000b142080d000000004ad5090eJ�e0008aD��], retry the api.
21:29:01 INFO WechatGateway sendShort() failed for error: sendShort receive unknown package: [126] 7e0ffffffff300000000b142080d000000004ad5090eJ�e0008aD��], retry the api.
21:29:02 INFO WechatGateway sendShort() failed for error: sendShort receive unknown package: [126] 7e0ffffffff300000000b142080d000000004ad5090eJ�e0008aD��], retry the api.
21:29:02 INFO WechatGateway sendShort() failed for error: sendShort receive unknown package: [126] 7e0ffffffff300000000b142080d000000004ad5090eJ�e0008aD��], retry the api.
21:29:02 ERR WechatGateway Error happened when call api: GrpcSendMessage, params: {"Content":"","MsgSource":"","ToUserName":""}
J�ndShorD��]eive unknown package: [126] 7e0ffffffff300000000b142080d000000004ad5090e44e0008a0400000000 ~�����B
21:29:11 INFO PadproManager Connection has problem, reset self to recover the connection.
21:29:11 WARN PuppetPadpro startManager() manager.on(reset) for undefined. Restarting PuppetPadpro ... 
Contact<> logout
21:29:11 INFO Wechaty <Puppet#0<PuppetPadpro>()/PuppetPadpro#0> stop() v0.23.40 is stoping ...
21:29:11 WARN PuppetPadpro stop() is called on a OFF puppet. await ready(off) and return.

## 3. 复现的步骤 (或者解决的步骤)

**这一部分非常重要，如果你无法给出复现步骤，我们也很难提供相应的解决办法：**
例子:
- 第一步：重新扫码
- 第二步：正常发出消息10条左右
- 第三步：报错如下
10:06:43 INFO PadproManager Connection has problem, reset self to recover the connection.
10:06:43 WARN PuppetPadpro startManager() manager.on(reset) for undefined. Restarting PuppetPadpro ... 
10:06:43 INFO CacheManager release()
10:06:43 WARN PadproManager releaseQueue() subscriptions have been released.
10:06:43 WARN PadproManager releaseQueue() queues have been released.
10:06:43 INFO CacheManager release()
10:06:43 WARN CacheManager init() CacheManager not exist, no need to release it.
10:06:43 INFO WechatGateway stop()
10:06:43 INFO WechatGateway initLongSocket() connection to wechat long host server: long.weixin.qq.com ended.
10:06:43 INFO WechatGateway init(********, padpro.botorange.com:6666, undefined)
10:06:43 INFO WechatGateway initLongSocket() connection to wechat long host server: long.weixin.qq.com closed.
10:06:44 INFO WechatGateway initLongSocket() connected to wechat long host server: long.weixin.qq.com.
10:06:44 WARN PadproGrpc GrpcAutoLogin() login failed with result: {"longHost":"long.weixin.qq.com","shortHost":"short.weixin.qq.com","status":-100,"userName":"wxid_nqluo69ukaw221"}
10:06:44 WARN PadproManager tryAutoLogin receive unknown api call status, if you keep seeing this status, please file an issue with detailed log to us, we will fix it ASAP.
Scan QR Code to login: 0

## 4. 期待程序运行的结果
请用精简的语言描述你期待运行的结果

## 5. 程序实际运行的结果
请用精简的语言或者截图描述实际运行的结果（**请不要在这里粘贴日志截图**）

## 6. 完整的日志信息：
通过`WECHATY_LOG=silly`设定环境变量，将设置log 等级为 silly，获取最详细的日志信息（默认log 等级为 info）
**请提供完整的日志信息(不要只提供部分的日志截图，请复制粘贴日志内容！)**
<details>
<summary>
Show Logs
</summary>

```shell
$ WECHATY_LOG=silly node yourbot.js

```

</details>

## 7. 其他信息
有相关bug 的背景信息，可以在这里说明

[bug]

**Comment by windmemory** [comment]
能否麻烦提供一个复现步骤？

**Comment by 763147503** [comment]
> 能否麻烦提供一个复现步骤？

我这样写可以吗

**Comment by windmemory** [comment]
This should be fixed with 0.0.51, please update your padpro version and observe whether this issue still happens.

**Comment by windmemory** [comment]
Close this issue since no response in two days. Please reopen if you sees this again.
